src/v2/Apps/Artist/Routes/Consign/Components/ArtistConsignPageViews.tsx

```diff
@@ -11,17 +11,13 @@ interface ArtistConsignPageViewsProps {
 
 export const ArtistConsignPageViews: React.FC<ArtistConsignPageViewsProps> = props => {
   const {
-    artist: {
-      name,
-      targetSupply: {
-        // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-        microfunnel: {
-          metadata: { roundedViews, roundedUniqueVisitors },
-        },
-      },
-    },
+    artist: { name, targetSupply },
   } = props
 
+  const roundedViews = targetSupply?.microfunnel?.metadata?.roundedViews || "0"
+  const roundedUniqueVisitors =
+    targetSupply?.microfunnel?.metadata?.roundedUniqueVisitors || "0"
+
```

Comment:
is it ok to set those to `"0"` in case they are `null`? 

I went for this approach because below, we are rendering these values in a string, for example "...works have received more than {roundedViews} views on Artsy this month"

Reply:
Looks correct to me.

Reply:
But the fact that a non-nullable number was typed as a nullable string in Metaphysics is super depressing...

Reply:
yeah the more I work on this the more I realise that there's work to be done in MP in order to get rid of those `// @ts-expect-error`s in Force